### DISCUSSION TITLE
Deep linking use case with tabs

### DISCUSSION THREAD

**Discussion by jonasgroendahl** [react-navigation/react-navigation#10861] [Q&A] [answered]
Hi!

Considering following setup:

```
- Root 
    - ModalScreen
    - Chat (I put the chat here in the hierarchy to hide the bottom tabs when chat screen is open)
    - BottomTabs
       - Home
```

Is it possible to deep link to the Chat Screen but also "load the bottom tab navigation"? So it goes like this:

-> BottomTabs -> Home -> Chat?

So instead of opening directly on the chat screen, it should have the bottom tabs -> home as part of state (routes), so you can "go back" to home from the chat screen

**Comment by satya164** [comment, marked as answer]
You can return a custom state using the `getStateFromPath` option in the linking config if you want to load a specific set of screens.

But also consider that as a user you click on some deep link that opens the app, and then after you're done, you want to go back to what you were doing - but now you'd have to press back 3 times to actually exit the app - which isn't a good user experience.
